tests/run.py

```diff
@@ -1399,15 +1412,17 @@ def main():
         if('golden_output' in test_params_dict):
             golden_stdout, golden_md5sum = get_file_content_and_md5sum(test_params_dict['golden_output']);
             if(golden_md5sum != md5sum_hash_str):
+	        is_error = True
                 if(bcftools_path is None):
                     sys.stderr.write('Did not find bcftools in path. If you are able to add that to the path, we can get a more precise diff\n')
```

Comment:
There was no error raised in this if statement. Tests were reporting error but the script wasn't failing. Can whoever modified this script with extra if-else statements take another look to see if there are similar issues.
See errors in the log of a previous PR:  https://api.travis-ci.org/v3/job/672595164/log.txt
Fortunately, no serious errors went undetected.

Reply:
Whoops. That was me I think. I'll take a look, and maybe try to add bcftools install to the travis build to see if that helps the travis build.